denoise/richardson_lucy/main.py

```diff
@@ -0,0 +1,17 @@
+if __name__ == "__main__":
+    from skimage import color, data, restoration
+    import richardson_lucy as rl
+    import numpy as np
+    import tensorflow as tf
+    camera = color.rgb2gray(data.camera())
+    from scipy.signal import convolve2d
+    import matplotlib.pyplot as plt
+
+    psf = np.ones((5, 5)) / 25
+    camera = convolve2d(camera, psf, 'same')
```

Comment:
Comments missing. Header block is also advisable in the beginning of the file to state purpose, author etc.